third_party/blink/renderer/brave_font_whitelist.cc

```diff
@@ -12,858 +12,828 @@ namespace brave {
-                                      "sans-serif-thin",
-                                      "source sans pro",
-                                      "source-sans-pro-semi-bold",
-                                      "tahoma",
-                                      "times",
-                                      "times new roman",
-                                      "verdana"});
+constexpr auto kFontWhitelist =
+    base::MakeFixedFlatSet<std::string_view>({"androidclock",
+                                              "arial",
+                                              "baskerville",
+                                              "carrois gothic",
+                                              "coming soon",
+                                              "courier",
+                                              "courier new",
+                                              "cutive mono",
+                                              "dancing script",
+                                              "droid sans",
+                                              "droid sans mono",
+                                              "erif-bold",
+                                              "fantasy",
+                                              "georgia",
+                                              "goudy",
+                                              "helvetica",
+                                              "itc stone serif",
+                                              "monaco",
+                                              "noto color emoji",
+                                              "noto kufi arabic",
+                                              "noto naskh arabic",
+                                              "noto nastaliq urdu",
+                                              "noto sans",
+                                              "noto serif",
+                                              "palatino",
+                                              "roboto",
+                                              "roboto static",
+                                              "sans-serif-black",
+                                              "sans-serif-condensed-light",
+                                              "sans-serif-condensed-medium",
+                                              "sans-serif-light",
+                                              "sans-serif-medium",
+                                              "sans-serif-monospace",
+                                              "sans-serif-thin",
+                                              "source sans pro",
+                                              "source-sans-pro-semi-bold",
+                                              "tahoma",
+                                              "times",
+                                              "times new roman",
+                                              "verdana"});
```

Comment:
just do a mass replacement in this file: `"});` -> `",});` and run clang-format.

Reply:
done